cranelift/fuzzgen/src/function_generator.rs

```diff
@@ -1266,7 +1259,7 @@ const OPCODE_SIGNATURES: &[OpcodeSignature] = &[
     (Opcode::Fma, &[F32, F32, F32], &[F32]),
     (Opcode::Fma, &[F64, F64, F64], &[F64]),
     (Opcode::Fma, &[F32X4, F32X4, F32X4], &[F32X4]),
-    (Opcode::Fma, &[F64X2, F64X2, F64X4], &[F64X2]),
+    (Opcode::Fma, &[F64X2, F64X2, F64X2], &[F64X2]),
```

Comment:
I was initially surprised as to how this didn't crash, but I guess we never generate `F64X4`'s so we never selected this!